docs/sidebars.js

```diff
@@ -74,6 +74,7 @@ const sidebars = {
       items: [
         "mask-filters",
         "color-filters",
+        "image-filters/drop-shadows",
```

Comment:
I would like to reorganize/clean the documentation categories. However, it might be better to do it in a separate PR.

Reply:
Agree!